vkontakte/content/contents/code/vkontakte.js

```diff
@@ -1,398 +1,291 @@
-/* === This file is part of Tomahawk Player - <http://tomahawk-player.org> ===
+/* vk.com resolver for Tomahawk.
  *
- *   Copyright 2011, Krzysztof Klinikowski <kkszysiu@gmail.com>
+ * Written in 2015 by Anton Romanov
+ * API documentation: https://vk.com/dev
  *
- *   Tomahawk is free software: you can redistribute it and/or modify
- *   it under the terms of the GNU General Public License as published by
- *   the Free Software Foundation, either version 3 of the License, or
- *   (at your option) any later version.
+ * To the extent possible under law, the author(s) have dedicated all
+ * copyright and related and neighboring rights to this software to
+ * the public domain worldwide. This software is distributed without
+ * any warranty.
  *
- *   Tomahawk is distributed in the hope that it will be useful,
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- *   GNU General Public License for more details.
- *
- *   You should have received a copy of the GNU General Public License
- *   along with Tomahawk. If not, see <http://www.gnu.org/licenses/>.
+ * You should have received a copy of the CC0 Public Domain Dedication
+ * along with this software. If not, see:
+ * http://creativecommons.org/publicdomain/zero/1.0/
```

Comment:
I'd prefer a BSD/MIT license over creative commons for code, easier to handle license combinations with those